modules/installation-creating-gcp-bootstrap.adoc

```diff
@@ -105,3 +134,21 @@ $ gcloud compute target-pools add-instances \
 $ gcloud compute target-pools add-instances \
     ${INFRA_ID}-ign-target-pool --instances-zone="${ZONE_0}" --instances=${INFRA_ID}-bootstrap
 ----
+
```

Comment:
Adding bootstrap to external load balancer target pool is not required. So please remove step 7:


The templates do not manage load balancer membership due to limitations of Deployment Manager, so you must add the bootstrap machine manually:

$ gcloud compute target-pools add-instances \
    ${INFRA_ID}-api-target-pool --instances-zone="${ZONE_0}" --instances=${INFRA_ID}-bootstrap
$ gcloud compute target-pools add-instances \
    ${INFRA_ID}-ign-target-pool --instances-zone="${ZONE_0}" --instances=${INFRA_ID}-bootstrap